src/SprykerEco/Zed/Braintree/Communication/Plugin/Checkout/BraintreeCheckoutPostSavePlugin.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Braintree\Communication\Plugin\Checkout;
+
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Checkout\Dependency\Plugin\CheckoutPostSaveHookInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin as BaseAbstractPlugin;
+
+/**
+ * @method \SprykerEco\Zed\Braintree\Business\BraintreeFacadeInterface getFacade()
+ */
+class BraintreeCheckoutPostSavePlugin extends BaseAbstractPlugin implements CheckoutPostSaveHookInterface
+{
+    /**
+     * {@inheritDoc}
+     *
```

Comment:
Pls remove the empty line. Check everywhere.

Reply:
Updated.